e2e/paths/logIn_test.js

```diff
@@ -1,6 +0,0 @@
-Feature('E2E Local Authority Happy Path');
-
-Scenario('Login as Local Authority and create case', (I, config) => {
```

Comment:
It would be good to exclude that change from this PR as it is not related to service scaffolding.